curriculum/challenges/_meta/build-an-arithmetic-formatter-project/meta.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Build an Arithmetic Formatter Project",
+  "isUpcomingChange": false,
+  "usesMultifileEditor": true,
+  "hasEditableBoundaries": true,
```

Comment:
By saying the projects have editable boundaries, you are telling the editor to expect the instructions in the editor jaws.